tests/osv_agent_test.py

```diff
@@ -209,3 +209,83 @@ def testAgentOSV_whenFingerprintMessage_processMessage(
         == "Use of Outdated Vulnerable Component: lodash@4.17.21: CVE-2021-23337"
     )
     assert agent_mock[6].data["risk_rating"] == "HIGH"
+
+
+def testAgentOSV_whenRiskLower_doesNotCrash(
```

Comment:
```suggestion
def testAgentOSV_whenRiskLowerCase_doesNotCrash(
```

saying lower implies the risk itself is lower